Makefile

```diff
@@ -16,8 +16,12 @@ clean:
 	rm -rf bin
 
 .PHONY: build-%
+# Attached .exe tail when making wincred file
+build-wincred: # build-wincred only
+	go build -trimpath -ldflags="$(GO_LDFLAGS) -X ${GO_PKG}/credentials.Name=docker-credential-wincred" -o "$(DESTDIR)/docker-credential-wincred.exe" ./wincred/cmd/
+
 build-%: # build, can be one of build-osxkeychain build-pass build-secretservice build-wincred
-	go build -trimpath -ldflags="$(GO_LDFLAGS) -X ${GO_PKG}/credentials.Name=docker-credential-$*" -o "$(DESTDIR)/docker-credential-$*" ./$*/cmd/
+	go build -trimpath -ldflags="$(GO_LDFLAGS) -X ${GO_PKG}/credentials.Name=docker-credential-$*" -o "$(DESTDIR)/docker-credential-$*.exe" ./$*/cmd/
```

Comment:
The .exe wouldn't be needed here, would it? As this entry would be used for anything other than build-wincred

Reply:
Yes it wouldn't be needed in that line. I removed it